draft-ietf-rats-msg-wrap.md

```diff
@@ -104,31 +104,30 @@ In addition, a Media Type and a CoAP Content-Format are defined for transporting
 
 # Introduction
 
-The RATS architecture defines a handful of conceptual messages
+The IETF Remote ATtestation procedureS (RATS) architecture defines a handful of conceptual messages
 (see {{Section 8 of -rats-arch}}), such as Evidence and Attestation Results.
 Each conceptual message can have multiple claims encoding and serialization
 formats ({{Section 9 of -rats-arch}}). Throughout their lifetime, RATS
 conceptual messages are typically transported over different protocols.
 For example,
 
-- EAT {{-rats-eat}} Evidence in a "background check" topological
-arrangement first flows from Attester to Relying Party, and then from Relying
-Party to Verifier, over separate protocol legs.
-
-- Attestation Results for Secure Interactions (AR4SI) {{-rats-ar4si}} payloads in
-"passport" mode would be sent by the Verifier to the Attester and then, at a later
-point in time and over a different channel, from the Attester to the Relying Party.
-
-It is desirable to reuse any typing information associated with the messages
-across such protocol boundaries to minimize the cost associated with
-type registrations and maximize interoperability. With the CMW format described
-in this document, protocol designers do not need to update protocol specifications
-to support different conceptual messages. This approach reduces the implementation
-effort for developers to support different attestation technologies. For example,
-an implementer of a Relying Party application does not need to parse
-attestation-related conceptual messages, such as different Evidence formats,
-but can instead utilize the CMW format to be agnostic to the attestation
-technology.
+- In a "background check" topology, EAT Evidence {{-rats-eat}} first flows from
+the Attester to the Relying Party and then from the Relying Party to the Verifier,
+each leg following a separate protocol path.
+
+- In "passport" mode, Attestation Results for Secure Interactions (AR4SI) payloads
```

Comment:
```suggestion
- In a "passport" topology, Attestation Results for Secure Interactions (AR4SI) payloads
```

Reply:
AR4SI is exemplary, and without some additional adaptation, AR4SI isn't even a message that can be sent. Better to say? "In a 'passport' topology, an attestation result payload (e.g., AR4SI [I-D.ar4si]) ..."